2023-11-30 22:11:22,900 INFO    MainThread:5872 [wandb_setup.py:_flush():76] Current SDK version is 0.16.0
2023-11-30 22:11:22,900 INFO    MainThread:5872 [wandb_setup.py:_flush():76] Configure stats pid to 5872
2023-11-30 22:11:22,900 INFO    MainThread:5872 [wandb_setup.py:_flush():76] Loading settings from C:\Users\max44\.config\wandb\settings
2023-11-30 22:11:22,900 INFO    MainThread:5872 [wandb_setup.py:_flush():76] Loading settings from C:\Users\max44\OneDrive\GitHub\PINN-Project-02456\Notebooks\wandb\settings
2023-11-30 22:11:22,900 INFO    MainThread:5872 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-11-30 22:11:22,900 INFO    MainThread:5872 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-11-30 22:11:22,907 INFO    MainThread:5872 [wandb_init.py:_log_setup():524] Logging user logs to C:\Users\max44\OneDrive\GitHub\PINN-Project-02456\Notebooks\wandb\run-20231130_221122-jdbrz5cq\logs\debug.log
2023-11-30 22:11:22,907 INFO    MainThread:5872 [wandb_init.py:_log_setup():525] Logging internal logs to C:\Users\max44\OneDrive\GitHub\PINN-Project-02456\Notebooks\wandb\run-20231130_221122-jdbrz5cq\logs\debug-internal.log
2023-11-30 22:11:22,909 INFO    MainThread:5872 [wandb_init.py:init():564] calling init triggers
2023-11-30 22:11:22,909 INFO    MainThread:5872 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0003, 'architecture': 'FFNN (8*32)', 'epochs': 1000, 'ODE loss': 'yes', 'IC loss': 'yes', 'Data points': 'true'}
2023-11-30 22:11:22,910 INFO    MainThread:5872 [wandb_init.py:init():589] re-initializing run, found existing run on stack: ssm9ozh2
2023-11-30 22:11:22,921 INFO    MainThread:5872 [wandb_run.py:_finish():1962] finishing run dl_pinn/SIR_PINN_test/ssm9ozh2
2023-11-30 22:11:22,922 INFO    MainThread:5872 [jupyter.py:save_history():445] not saving jupyter history
2023-11-30 22:11:22,922 INFO    MainThread:5872 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-11-30 22:11:22,923 INFO    MainThread:5872 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2023-11-30 22:11:22,923 INFO    MainThread:5872 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2023-11-30 22:11:22,924 INFO    MainThread:5872 [wandb_run.py:_restore():2185] restore
2023-11-30 22:11:22,924 INFO    MainThread:5872 [wandb_run.py:_restore():2191] restore done
2023-11-30 22:11:29,859 INFO    MainThread:5872 [wandb_run.py:_footer_history_summary_info():3760] rendering history
2023-11-30 22:11:29,860 INFO    MainThread:5872 [wandb_run.py:_footer_history_summary_info():3792] rendering summary
2023-11-30 22:11:29,877 INFO    MainThread:5872 [wandb_run.py:_footer_sync_info():3719] logging synced files
2023-11-30 22:11:30,079 INFO    MainThread:5872 [wandb_init.py:init():614] starting backend
2023-11-30 22:11:30,079 INFO    MainThread:5872 [wandb_init.py:init():618] setting up manager
2023-11-30 22:11:30,085 INFO    MainThread:5872 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2023-11-30 22:11:30,088 INFO    MainThread:5872 [wandb_init.py:init():624] backend started and connected
2023-11-30 22:11:30,152 INFO    MainThread:5872 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2023-11-30 22:11:30,152 INFO    MainThread:5872 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-11-30 22:11:30,152 INFO    MainThread:5872 [wandb_init.py:init():716] updated telemetry
2023-11-30 22:11:30,278 INFO    MainThread:5872 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2023-11-30 22:11:31,667 INFO    MainThread:5872 [wandb_run.py:_on_init():2254] communicating current version
2023-11-30 22:11:31,797 INFO    MainThread:5872 [wandb_run.py:_on_init():2263] got version response 
2023-11-30 22:11:31,797 INFO    MainThread:5872 [wandb_init.py:init():800] starting run threads in backend
2023-11-30 22:12:02,039 INFO    MainThread:5872 [wandb_run.py:_console_start():2233] atexit reg
2023-11-30 22:12:02,044 INFO    MainThread:5872 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2023-11-30 22:12:02,044 INFO    MainThread:5872 [wandb_run.py:_redirect():2153] Wrapping output streams.
2023-11-30 22:12:02,045 INFO    MainThread:5872 [wandb_run.py:_redirect():2178] Redirects installed.
2023-11-30 22:12:02,047 INFO    MainThread:5872 [wandb_init.py:init():841] run started, returning control to user process
2023-11-30 22:12:02,049 INFO    MainThread:5872 [wandb_watch.py:watch():51] Watching
2023-11-30 22:14:16,937 INFO    MainThread:5872 [wandb_run.py:_finish():1962] finishing run dl_pinn/SIR_PINN_test/jdbrz5cq
2023-11-30 22:14:16,938 INFO    MainThread:5872 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2023-11-30 22:14:16,940 INFO    MainThread:5872 [wandb_run.py:_restore():2185] restore
2023-11-30 22:14:16,940 INFO    MainThread:5872 [wandb_run.py:_restore():2191] restore done
2023-11-30 22:14:25,183 INFO    MainThread:5872 [wandb_run.py:_footer_history_summary_info():3760] rendering history
2023-11-30 22:14:25,185 INFO    MainThread:5872 [wandb_run.py:_footer_history_summary_info():3792] rendering summary
2023-11-30 22:14:25,212 INFO    MainThread:5872 [wandb_run.py:_footer_sync_info():3719] logging synced files
